planex/link.py

```diff
@@ -27,6 +27,11 @@ def schema_version(self):
         # Default to v1 if not present, for legacy lnks
         return int(self.link.get('SchemaVersion', 1))
 
+    @property
+    def ignore_autosetup(self):
+        """Return the ignore autosetup value"""
+        return bool(self.link.get('IgnoreAutosetup', False))
```

Comment:
Is the `bool()` wise? If the input file has

```
"IgnoreAutosetup": "False"
```

then it will silently become `True`

Reply:
You are right. For some unclear reasons I was convinced that it would have evaluated the `"False"` into `False`, that is clearly false!